staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -748,6 +756,16 @@ func (p *awsSDKProvider) getCrossRequestRetryDelay(regionName string) *CrossRequ
 	return delayer
 }
 
+// SetInformers sets up informer-fed caches for aws lib to leverage Kubernetes API for caching
+func (c *Cloud) SetInformers(informerFactory informers.SharedInformerFactory) {
+	klog.Infof("Setting up informers for Cloud")
+	defer func() {
+		klog.Infof("Finished setting up informers for Cloud")
+	}()
```

Comment:
Why are we using this defer?

Reply:
this is to log a line after informers are setup. Is adding this line at the end of the function a better practice?

Reply:
anyway, felt like it's redundant as well for such a setup-routine, removed.